src/main/java/seedu/address/model/Model.java

```diff
@@ -85,6 +85,12 @@ public interface Model {
      */
     void setPerson(Person target, Person editedPerson);
 
+    /**
+     * Check if any of the already exist for the person.
+     * {@code person} must already exist in the address book.
+     */
+    Boolean tagExists(Person target, Set<Tag> tags);
```

Comment:
Similar to what Jianbing said I believe this is returning a Boolean wrapper, not the primitive one!